File: source/code/ruby/authentication/mfa_create_and_challenge_req.rb
factor = account.create_factor(:sms,
                               phone: { number: '267-555-5555',
                                        name: 'Rspec test phone',
                                        description: 'This is a testing phone number' },
                               challenge: { message: 'Welcome to the Example! Your authorization code is ' })

File: source/code/nodejs/authentication/mfa_create_and_challenge_req.js
var factor = {
  type: "sms",
  challenge: { // An optional parameter
    message: "Welcome to the Example! Your authorization code is ${code}"
  },
  phone: {
    number: '+12675555555'
  }
};

var account; // A previously fetched Account object

account.createFactor(factor, function(err, smsFactor) {
  if (err) {
    return console.log(err);
  }

  console.log(smsFactor);
});
